src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,52 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+public class Ui {
+    private final String HORIZONTAL_LINE = "    _______________________________________________________________";
+    public void printList(TaskList taskList) {
+        System.out.println("    Here are the tasks in your list:");
+        taskList.printTaskList();
+    }
+
+    public void hello() {
+        System.out.println(HORIZONTAL_LINE);
```

Comment:
Since you have a `printHorizontalLine()` method, maybe you can use that instead.